storescp/src/main.rs

```diff
@@ -104,6 +108,20 @@ fn create_cecho_response(message_id: u16) -> InMemDicomObject<StandardDataDictio
 
 fn main() {
     let app = App::parse();
+    tracing::subscriber::set_global_default(
+        tracing_subscriber::FmtSubscriber::builder()
+            .with_max_level(Level::INFO)
+            .with_env_filter(
+                EnvFilter::from_default_env()
+                    .add_directive("app_common=info".parse().unwrap())
+                    .add_directive(if app.verbose { "storescp=debug".parse().unwrap() } else { "storescp=info".parse().unwrap() })
```

Comment:
Without the full crate name, no logs would appear.
```suggestion
                    .add_directive("dicom_app_common=info".parse().unwrap())
                    .add_directive(if app.verbose { "dicom_storescp=debug".parse().unwrap() } else { "dicom_storescp=info".parse().unwrap() })
```